api/tests/Api/Invitations/FindInvitationTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace App\Tests\Api\Invitations;
+
+use App\Entity\CampCollaboration;
+use App\Tests\Api\ECampApiTestCase;
+
+/**
+ * @internal
+ */
+class FindInvitationTest extends ECampApiTestCase {
+    const OPTIONS = ['headers' => ['Accept' => 'application/hal+json']];
```

Comment:
Ist es nötig die options immer mitzugeben? Bei den anderen tests musste ich das glaube ich nicht, weil ich HAL in api platform als höchste priorität konfiguriert habe

Reply:
War hier glaube ich nötig, da sonst jsonld zurück kam.
Man könnte es natürlich wieder als erste Priorität definieren, hätte dann aber duplikation.
Ich würde es für diesen PR mal so lassen

Reply:
Du hast recht, man muss createBasicClient machen, das macht das schon für einem.